packages/next/src/client/components/react-dev-overlay/internal/helpers/get-error-by-type.ts

```diff
@@ -28,11 +28,19 @@ export async function getErrorByType(
         id,
         runtime: true,
         error: event.reason,
-        frames: await getOriginalStackFrames(
-          event.frames,
-          getErrorSource(event.reason),
-          isAppDir
-        ),
+        // createMemoizedPromise dedups calls to getOriginalStackFrames
+        frames: createMemoizedPromise(async () => {
+          try {
+            return await getOriginalStackFrames(
+              event.frames,
+              getErrorSource(event.reason),
+              isAppDir
+            )
+          } catch (e) {
+            console.error(e)
+            return []
+          }
```

Comment:
Why do we catch this now? `console.error` would create another issue in the overlay but it probably shouldn't?